test/projectDataProvider.test.ts

```diff
@@ -127,8 +127,8 @@ suite('projectDataProvider Modules', () => {
         }
         let stubExec = sandbox.stub(child_process, 'exec').yields(null, 'success', 'success');
         let stubExistsSync = sandbox.stub(fs, 'existsSync').returns(true);
-        ProjectDataProvider.getDependencyVersion(vscodeRootpath, (cb) => {
-            expect(cb).equals(true);
+        await ProjectDataProvider.getDependencyVersion(vscodeRootpath).then((data) => {
```

Comment:
why do we need then cb here we can directly check the value of the function

Reply:
you are using `await` here to resolve the promise right?

Reply:
yes have made used use of `then` to handle it and `await` is used as i want it to execute before we test below
`
expect(stubExistsSync).callCount(2);
 expect(stubExec).calledOnce;
`

Reply:
but now you can remove then cb.
```
const result = await ProjectDataProvider.getDependencyVersion(vscodeRootpath)
expect(result).calledOnceWith(true);
```

Reply:
ah got it, thanks !!